cmd/ui/src/views/Explore/ExploreSearch/ExploreSearch.tsx

```diff
@@ -140,7 +140,9 @@ const ExploreSearch: React.FC = () => {
             params.exploreSearchTab = 'pathfinding';
         }
         if (tab === 'cypher') {
-            params.searchType = 'cypher';
+            if (cypherSearchState.cypherQuery) {
+                params.searchType = 'cypher';
+            }
```

Comment:
see BED-5663: When changing the exploreSearchTab, we should only update the searchType when values for the searchType exist. Right now, when selecting the cypher tab, we always set the searchTab to cypher, even if a cypherSearch is missing.